contrib/CONTRIBUTING.md

```diff
@@ -34,6 +37,34 @@ Additionally, note that the core implementation of Bittensor consists of two sep
 
 Supplemental, yet necessary repositories are [openvalidators](https://github.com/opentensor/validators) and [openminers](https://github.com/opentensor/miners) which contain Bittensor Validators and Miners (respectively) designed by the OpenTensor Foundation team and open-sourced for the community to use.
 
+## Getting Started
+New contributors are very welcome and needed.
+Reviewing and testing is highly valued and the most effective way you can contribute as a new contributor. It also will teach you much more about the code and process than opening pull requests. 
+
+Before you start contributing, familiarize yourself with the Bittensor Core build system and tests. Refer to the documentation in the repository on how to build Bittensor core and how to run the unit tests, functional tests.
+
+There are many open issues of varying difficulty waiting to be fixed. If you're looking for somewhere to start contributing, check out the [good first issue](https://github.com/opentensor/bittensor/labels/good%20first%20issue) list or changes that are up for grabs. Some of them might no longer be applicable. So if you are interested, but unsure, you might want to leave a comment on the issue first.
+### Good First Issue Label
+The purpose of the good first issue label is to highlight which issues are suitable for a new contributor without a deep understanding of the codebase.
+
+However, good first issues can be solved by anyone. If they remain unsolved for a longer time, a frequent contributor might address them.
+
+You do not need to request permission to start working on an issue. However, you are encouraged to leave a comment if you are planning to work on it. This will help other contributors monitor which issues are actively being addressed and is also an effective way to request assistance if and when you need it.
+### Beginner and Help-wanted Issues Label
+You can start by looking through these `beginner` and `help-wanted` issues:
+
+* [Beginner issues](https://github.com/opentensor/bittensor/labels/beginner) - issues which should only require a few lines of code, and a test or two.
+* [Help wanted issues](https://github.com/opentensor/bittensor/labels/help%20wanted) - issues which should be a bit more involved than `beginner` issues.
+
+
+## Communication Channels
+Most communication about Bittensor development happens on Discord channel.
+Here's the link of Discord community.
+https://discord.com/channels/799672011265015819/799672011814862902
```

Comment:
It'd best practice not to leave full hyperlinks in documents like this.

```suggestion
[Bittensor Discord](https://discord.com/channels/799672011265015819/799672011814862902)
```

Reply:
Thanks for your advice.